### PR TITLE
Introduce black ruff isort in pipeline

### PR DESCRIPTION
**Pull request by dbalucas**
This pull request should introduce code formater black with special configuration of 120 lines to the code. As well as extensions to the CI.yml pipeline which check whether the code was properly standardized and ordered with isort.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[tool.black]
+line-length = 120
+target-version = ['py311']
+
+[tool.isort]
+line_length = 120
```
Comment (raynigon) [inline pyproject.toml:6]: Missing new line

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,7 +3,58 @@ name: CI
...
 jobs:
-  lint:
+  lint_black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```
Comment (raynigon) [inline .github/workflows/ci.yml:13]: Use Environment variable

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -3,7 +3,58 @@ name: CI
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install black
+          pip install -r requirements.txt
+      - uses: psf/black@stable
+        with:
+            options: "--check --verbose"
+            src: "./src"
+            version: "~= 22.0"
```
Comment (raynigon) [inline .github/workflows/ci.yml:23]: Use Environment variable

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -3,7 +3,58 @@ name: CI
...
+  lint_ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```
Comment (raynigon) [inline .github/workflows/ci.yml:31]: Use Environment variable

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -3,7 +3,58 @@ name: CI
...
+          # stop the build if there are Python syntax errors or undefined names
+          ruff --format=github --select=E9,F63,F7,F82 --target-version=py311 .
+  lint_isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```
Comment (raynigon) [inline .github/workflows/ci.yml:48]: Use Environment variable

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -3,7 +3,58 @@ name: CI
...
+          pip install -r requirements.txt
+      - name: Lint with ruff
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          ruff --format=github --select=E9,F63,F7,F82 --target-version=py311 .
+  lint_isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
```
Comment (raynigon) [inline .github/workflows/ci.yml:45]: Use Environment variable

File: .github/workflows/ci.yml [hunk 7]
```diff
@@ -3,7 +3,58 @@ name: CI
...
+      - uses: psf/black@stable
+        with:
+            options: "--check --verbose"
+            src: "./src"
+            version: "~= 22.0"
+  lint_ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
```
Comment (raynigon) [inline .github/workflows/ci.yml:28]: Use Environment variable

**Review by raynigon — CHANGES_REQUESTED** [review]
Please use a variable for python versions:
https://docs.github.com/en/actions/learn-github-actions/variables

File: src/lennybot/config/__init__.py [hunk 8]
```diff
@@ -1,2 +1 @@
-from .config import (LennyBotActionConfig, LennyBotAppConfig,
-                     LennyBotCheckConfig, LennyBotConfig, LennyBotSourceConfig)
+from .config import LennyBotActionConfig, LennyBotAppConfig, LennyBotCheckConfig, LennyBotConfig, LennyBotSourceConfig
```
[the remark this answers is not in the document]
Comment (dbalucas) [reply src/lennybot/config/__init__.py:1]: @sonatype-lift ignore

File: src/lennybot/__init__.py [hunk 9]
```diff
@@ -27,24 +27,44 @@ def _find_config(args):
     if os.path.exists("config.yml"):
         return "config.yml"
     raise Exception(
-        "Configuration file not found, create 'config.yaml' or set LB_CONFIG_FILE to point to the config file")
+        "Configuration file not found, create 'config.yaml' or set LB_CONFIG_FILE to point to the config file"
```
[the remark this answers is not in the document]
Comment (dbalucas) [reply src/lennybot/__init__.py:30]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (dbalucas) [reply src/lennybot/__init__.py:30]: not needed our standard is 120 line_length

File: src/lennybot/lennybot.py [hunk 10]
```diff
@@ -82,6 +81,6 @@ def ci_finalize(self, plan: LennyBotPlan, result):
         self._repo.config_writer().set_value("user", "email", "lennybot@raynigon.com").release()
         self._repo.git.add(A=True)
         self._repo.git.commit(m=title)
-        self._repo.git.push('--set-upstream', 'origin', self._branch_name)
+        self._repo.git.push("--set-upstream", "origin", self._branch_name)
```
[the remark this answers is not in the document]
Comment (dbalucas) [reply src/lennybot/lennybot.py:84]: @sonatype-lift ignore

**Pull request merged**